src/content/docs/_zh-cn/blog/tauri-2-0-0-beta.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Tauri v2 Beta 版本发布公告
+date: 2024-02-02
+authors: [lucasfernog, kamaslau]
+excerpt: Tauri v2 已经准备好向稳定阶段进发啦！2.0.0-beta.0 版本已被发布。
+banner:
+  content: |
+    你正在阅读 Tauri 2.0 预发布版本的博客内容 -
+    <a href="https://tauri.app">前往 Tauri 1.0 站点</a>
+---
+
+![Hero Image](./tauri_2_0_0_alpha_0/header.png)
+
+Tauri v2 版本的相关工作已经进行了[一年有余](/blog/tauri-mobile-alpha)，如今已经准备好向稳定版本进发了！作为[工作路线图]（/blog/roadmap-to-tauri-2-0#beta）的里程碑事件，我们已经发布了[Tauri v2.0.0-beta.0](https://github.com/tauri-apps/tauri/releases/tag/tauri-v2.0.0-beta.0)。
+
+Tauri v2 版本引入了移动端支持，以及社群要求的多项新功能。让我们大致了解以下主要变化：
+
+## 移动端开发
+
+同时开发同一个应用的桌面端和移动端现在更加方便了。Tauri v2 版本在跨平台开发方面进展巨大——我们现已支持 Android 和 iOS。你可以将现有的桌面端程序无缝集成、分发为移动端，调用相应平台的原生 API，享受 Tauri 命令行工具带来的优秀开发体验。
+
+## 权限控制
+
+v1 版本的白名单机制在监管前端对非必要的 API 调用方面是一个好工具，然而它的配置有失粗放，且缺乏对多窗口的支持。2.0.0-beta.0 版本带来了基于调用控制列表（Access Control List）进行命令调用的新方式，实现了命令放行，以及为特定窗口（甚至远端 URL）进行作用域限制。
```

Comment:
```suggestion
v1 版本的白名单机制在监管前端对非必要的 API 调用方面是一个好工具，然而它的配置有些粗放，且缺乏对多窗口的支持。2.0.0-beta.0 版本带来了基于调用控制列表（Access Control List）进行命令调用的新方式，实现了命令放行，以及为特定窗口（甚至远端 URL）进行作用域限制。
```

Reply:
@simonhyll I'd stick with the original line. The word 有失 stated the coming statements as a blemish thus making it reasonable to be improved, whileas 有些, suggested by @veaba , just expresses a feeling.

Reply:
I don't know any Chinese, maybe @dklassic can offer some input? 😅

Reply:
有些粗放("somewhat rough") makes more sense structurally, as 有失 is generally used as "has wrongs" or "not [...] enough" both of which doesn't really make any sense.

As the original blog post stated "its configuration is not fine grained", I'd actually propose to stick to original post and change this to "然而它的配置不够精细".

Reply:
Whileas straightly translate word-by-word would be a safe solution, convert the context in a way which express its intention is in fact an accurate approach for a blog post that serves with marketing function rather than simply a tech document. Taking this into concern, the version 有失 or 有些 both indicates that Tauri is humble for its work and has the ambition of making improvements, and would seek for short bars to patch (even if it is already a revolutionary solution, to be speak of), among which 有失 shows more gumption. So in general I think picking 有失 is a good choice. @dklassic

Reply:
I agree that 有失 should come with words that targets good wills, so it should be replaced by 失之粗放.
While it's been stated clearly why a simple 不够精细 doesn't fit current context, I think it's more important to publish the article as a whole, so suggestion merged.